staging/src/k8s.io/apimachinery/pkg/util/remotecommand/constants.go

```diff
@@ -41,6 +45,9 @@ const (
 	// attachment/execution. It is the 4th version of the subprotocol and
 	// adds support for exit codes.
 	StreamProtocolV4Name = "v4.channel.k8s.io"
+
+	NonZeroExitCodeReason = metav1.StatusReason("NonZeroExitCode")
+	ExitCodeCauseType     = metav1.CauseType("ExitCode")
```

Comment:
they were defined in `pkg/kubelet/server/remotecommand/exec.go`. I had to move them as well cause they are used by client/remotecommand